src/css/angular-tooltips.css

```diff
@@ -36,44 +36,87 @@
   transition: opacity 0.2s linear;
   opacity: 1;
 }
+._720kb-tooltip-caret:before {
```

Comment:
There is something wrong with the caret style, i see tooltip content in lighter background color than caret, caret is more black , content is more grey